<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCategoryAndValue</class>
 <widget class="QDialog" name="SelectCategoryAndValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="category"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="value"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectCategoryAndValue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectCategoryAndValue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
